pkg/lib/localstore/ftsearch/ftsearchtantivy_test.go

```diff
@@ -15,58 +15,59 @@ import (
 	"github.com/anyproto/anytype-heart/core/wallet"
 )
 
-type fixture struct {
+type fixture2 struct {
 	ft   FTSearch
 	ta   *app.App
 	ctrl *gomock.Controller
 }
 
-func newFixture(path string, t *testing.T) *fixture {
-	ft := New()
+func newFixture2(path string, t *testing.T) *fixture2 {
+	ftsDir2 = ""
+	ft := TantivyNew()
 	ta := new(app.App)
 
 	ta.Register(wallet.NewWithRepoDirAndRandomKeys(path)).
 		Register(ft)
 
 	require.NoError(t, ta.Start(context.Background()))
-	return &fixture{
+	return &fixture2{
 		ft: ft,
 		ta: ta,
 	}
 }
 
-func TestNewFTSearch(t *testing.T) {
+func TestNewFTSearch2(t *testing.T) {
 	testCases := []struct {
 		name   string
 		tester func(t *testing.T, tmpDir string)
 	}{
 		{
 			name:   "assertSearch",
-			tester: assertSearch,
+			tester: assertSearch2,
 		},
 		{
 			name:   "assertThaiSubstrFound",
-			tester: assertThaiSubstrFound,
-		},
-		{
-			name:   "assertChineseFound",
-			tester: assertChineseFound,
+			tester: assertThaiSubstrFound2,
 		},
+		// {
```

Comment:
doesn't work and it is expected
https://github.com/yanyiwu/gojieba
test was taken from here
https://github.com/jiegec/tantivy-jieba
tested with this tokenizer and looks like this test proofs nothing

Reply:
Additionally, we can easily provide bilingual tokenization to improve search quality.
https://github.com/anyproto/tantivy-jpc/blob/77a12308911f0c746ce022e3c9124091219221ab/src/lib.rs#L514